build.gradle

```diff
@@ -155,7 +155,9 @@ subprojects {
         googleCloudBetaVersion = '0.54.0-beta'
         googleCloudGaVersion = '1.36.0'
         signalfxVersion = '0.0.39'
-        springVersion = '4.3.12.RELEASE'
+        springBootVersion = '1.5.15.RELEASE'
+	springCloudVersion = '1.3.4.RELEASE'
```

Comment:
spring boot 1 and 2 has very different sleuth. any reason this is using 1.5?
also, should this target core spring cloud sleuth version rather than a boot version?